doc/doc.md

```diff
@@ -166,8 +167,82 @@ export const settings = {
+
+#### Number
+
+| Prop         | Type                                                      | Default   | Description                                                                                                       |
+| ------------ | --------------------------------------------------------- | --------- | ----------------------------------------------------------------------------------------------------------------- |
+| type (\*)    | `"number"`                                                |           | Defines a param of type 'number'                                                                                  |
+| default (\*) | `number`                                                  |           | Default to this value.                                                                                            |
+| min          | `number`                                                  | undefined | Mininum acceptable value (required if `slider: true`)                                                             |
+| max          | `number`                                                  | undefined | Maximum acceptable value (required if `slider: true`)                                                             |
+| step         | `Step`                                                    | undefined | Step to increase of decrease value by                                                                             |
+| slider       | `boolean`                                                 | undefined | Wether to display input as as range slider. (if true, min and max are required)                                   |
+| editable     | `boolean`                                                 | true      | If false, disables the field in the UI.                                                                           |
+| options      | `['value']` or `[{label,value}]` or `{label: value, ...}` | undefined | If present, displays a dropdown with the provided options.                                                        |
+| validation   | `function`                                                | undefined | If present, executes function with the new value. Should return a string describing the error or null if no error |
+
+#### Boolean or Toggle
+
+| Prop         | Type        | Default   | Description                                                                                                       |
+| ------------ | ----------- | --------- | ----------------------------------------------------------------------------------------------------------------- |
+| type (\*)    | `"boolean"` |           | Defines a param of type 'boolean'                                                                                 |
+| default (\*) | `string`    |           | Default to this value.                                                                                            |
```

Comment:
Type should be boolean right?

Reply:
also while typing this i thought this should be optional ?. like no default == false ?